hypervisor/arch/x86/cpu.c

```diff
@@ -641,7 +641,7 @@ static void cpu_secondary_post(void)
 	timer_init();
 
 	/* Wait for boot processor to signal all secondary cores to continue */
-	pcpu_sync_sleep(&pcpu_sync, 0UL);
+	wait_sync_change(&pcpu_sync, PCPU_SYNC_RUN);
```

Comment:
I prefer to use 0UL here. 0 is something special, it becomes unclear if we use a MACRO

Reply:
Ok.